src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -111,74 +112,112 @@ private ExtImpIx parseImpExt(Imp imp) {
         }
     }
 
-    private static List<BidRequest> makeBidRequests(BidRequest bidRequest, Imp imp, String siteId) {
-        final Site modifiedSite = modifySite(bidRequest.getSite(), siteId);
+    private Imp modifyImp(Imp imp, ExtImpIx impExt) {
+        final UpdateResult<ObjectNode> impExtUpdateResult = modifyImpExt(imp, impExt);
+        final UpdateResult<Banner> bannerUpdateResult = modifyImpBanner(imp.getBanner());
```

Comment:
Update this to our default approach, returning null when not changed
Same for each method that returns `UpdateResult`